2012-12-19	Nicolas Setton  <setton@adacore.com>
	* configure: Deactivate use of mediaLib, not working on
	sparc64-solaris.

--- configure.orig	Fri Dec 14 11:35:19 2012
+++ configure	Fri Dec 14 11:37:48 2012
@@ -17729,7 +17729,7 @@
 
 if test $use_mlib = yes; then
 
-$as_echo "#define USE_MEDIALIB 1" >>confdefs.h
+# $as_echo "#define USE_MEDIALIB 1" >>confdefs.h
 
     MEDIA_LIB=-lmlib
 
